dev/reference/tools/data-migration/dm-portal.md

```diff
@@ -0,0 +1,235 @@
+---
+title: DM Portal 简介
+category: reference
+---
+
+# DM Portal 简介
+
+当前版本的 DM 的提供了丰富多样的功能特性，包括 [Table routing](/reference/tools/data-migration/features/overview.md#table-routing)，[Black & white table lists](/reference/tools/data-migration/features/overview.md#black-white-table-lists)，[Binlog event filter](/reference/tools/data-migration/features/overview.md#binlog-event-filter)，[Column mapping](/reference/tools/data-migration/features/overview.md#column-mapping) 等。但这些功能特性同时也增加了用户使用 DM 的复杂度，尤其在编写 [DM 任务配置](/reference/tools/data-migration/configure/task-configuration-file/) 时候。
+针对这个问题，我们提供了一个非常精简的小网页程序  DM Portal，能够帮助用户以可视化的方式去配置需要的同步任务，并且生成可以直接让 DM 直接执行的 task.yaml 文件。
+
+## 功能描述
+
+### 同步模式配置
+
+支持 DM 的三种同步模式：
+
+- 全量同步
+- 增量同步
+- All(全量+增量）
```

Comment:
```suggestion
- All（全量+增量）
```

Reply:
Fix in https://github.com/pingcap/docs-cn/pull/1679/commits/998f85400384502d92a0435fca5529c0d21b71fa